### PR TITLE
Update webpack-config-terra dependencies

### LINKED ISSUE #488: [webpack-config-terra] Bump postcss-custom-properties to resolve high severity vulnerabilities
# Bug Report

## Description
Installing alpha version of @cerner/webpack-config-terra results in high severity vulnerabilities. This should be resolved by upgraded postcss-custom-properties to v10.0.0.

https://www.npmjs.com/advisories/1550

Screen Shot 2020-11-23 at 1 52 59 PM

## Environment

* Component Name and Version: @cerner/webpack-config-terra 1.0.0-alpha.1

### PR DESCRIPTION
**Pull request by mjhenkes**
### Summary

This PR updates the dependencies we can, for webpack-config-terra.

The dependencies not updated are:
postcss: we can't use version 8 until an updated version of postcss-rtl is released.
terser-webpack-plugin: it requires webpack 5. Which is unfortunate because that dashes any hopes of supporting both webpack 4 and 5.
autoprefixer: the next version requires postcss version 8

### Additional Details

This has been tested locally with both old and new terra-dev-site and terra-frameworks WDIO test.

Closes #488 

@cerner/terra

[CONTRIBUTORS.md]: ../blob/main/CONTRIBUTORS.md
[License]: ../blob/main/LICENSE

### CODE REVIEW

File: packages/webpack-config-terra/webpack.config.js [hunk 1]
```diff
@@ -107,6 +107,7 @@ const defaultWebpackConfig = (env = {}, argv = {}, options = {}) => {
             /node_modules/.test(modulePath)
             && !/@cerner\/webpack-config-terra\/lib\/entry\/core-js/.test(modulePath)
           ), // exclude everything in node modules except our core-js entry point to allow consumers the ability to customize what polyfills get pulled in.
+          enforce: 'post',
```
Comment (mjhenkes) [inline packages/webpack-config-terra/webpack.config.js]: gonna be honest, i don't remember adding this. I need to investigate.

File: packages/webpack-config-terra/webpack.config.js [hunk 2]
```diff
@@ -119,9 +120,6 @@ const defaultWebpackConfig = (env = {}, argv = {}, options = {}) => {
           use: [
             {
               loader: MiniCssExtractPlugin.loader,
-              options: {
-                hmr: !production, // only enable hot module reloading in development
```
Comment (mjhenkes) [inline packages/webpack-config-terra/webpack.config.js:123]: Now done by default yay!

File: packages/webpack-config-terra/webpack.config.js [hunk 3]
```diff
@@ -137,14 +135,14 @@ const defaultWebpackConfig = (env = {}, argv = {}, options = {}) => {
             {
               loader: 'postcss-loader',
               options: {
-                // Add unique ident to prevent the loader from searching for a postcss.config file. See: https://github.com/postcss/postcss-loader#plugins
-                ident: 'postcss',
```
Comment (mjhenkes) [inline packages/webpack-config-terra/webpack.config.js:141]: No longer needed

File: packages/webpack-config-terra/webpack.config.js [hunk 4]
```diff
@@ -137,14 +135,14 @@ const defaultWebpackConfig = (env = {}, argv = {}, options = {}) => {
...
                 sourceMap,
-                plugins: [
```
Comment (mjhenkes) [inline packages/webpack-config-terra/webpack.config.js:143]: plugins are now nested in the postcssOptions object.

**Comment by mjhenkes** [comment]
Also, with these updates i think we should go to 1.0.0. I had been holding out hope that we could support both webpack 4 and 5 but 5 is looking like a major version since terser doesn't support both.

**Review by yuderekyu — APPROVED** [review]
want to make those verification branches into prs?

**Comment by mjhenkes** [comment]
@yuderekyu it's harder now that this is in a mono repo and we can't reference a git link, but I'll put one up for Framework using the generated tar from this package.
https://github.com/cerner/terra-framework/pull/1333

**Review by StephenEsser — APPROVED** [review]

**Pull request merged**